runtime/storage/cache/basic.go

```diff
@@ -309,8 +309,8 @@ func (s *IntKeyspace[K]) Increment(ctx context.Context, key K, delta int64) (new
 // but typically you want to use the Incr method for that.
```

Comment:
Saw that you reference them here as `Incr`/`Decr` as well. (In the other docs as well)

Reply:
Thanks, you have an amazing eye for catching issues!

Reply:
Was maybe a bit to interested in this feature and read the docs quite closely😅